src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -123,13 +123,16 @@
     private boolean usageRestricted;
 
     private int maxRequestsPerHost;
+    private static final boolean disableNoDelayProvisioning = Boolean.valueOf(
+            System.getProperty("io.jenkins.plugins.kubernetes.disableNoDelayProvisioning"));
```

Comment:
this namespace isn't used by the plugin, would be more consistent to use the package name. Or just move the new class to a new package `io.jenkins.plugins.kubernetes` ?